app/javascript/src/components/Dashboard/Tasks/taskList.jsx

```diff
@@ -1,14 +1,42 @@
 import React from "react";
-import { Checkbox, Badge, Avatar } from "neetoui";
+import { Checkbox, Badge, Avatar, Tooltip } from "neetoui";
+import deleteBtn from "images/DeleteButton";
+import editBtn from "images/EditButton";
 
 export default function TaskList({
   selectedTaskIds,
   setSelectedTaskIds,
+  setShowDeleteAlert,
   tasks = [],
 }) {
+  const handleDelete = taskIds => {
+    setSelectedTaskIds([taskIds]);
+    setShowDeleteAlert(true);
+  };
+
+  const deleteButton = taskid => {
+    return (
+      <Tooltip content="Delete Task" position="bottom">
+        <div onClick={() => handleDelete(taskid)}>
+          <img src={deleteBtn} alt="deletebutton" />
+        </div>
+      </Tooltip>
+    );
+  };
+
+  const editButton = () => {
+    return (
+      <Tooltip className="mx-2" content="Edit Task" position="bottom">
+        <div>
+          <img src={editBtn} alt="edit button" />
+        </div>
+      </Tooltip>
+    );
+  };
+
   return (
     <div className="w-full px-14">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--actions">
```

Comment:
Please add the className `nui-table--hover` here so that each row on hover shows a hover background effect. And please remove the custom tailwind className that you've used for the same. 

Also, please try adding className `nui-table--avatar`. This would fix the elongation of the avatar.